extensions/smallrye-fault-tolerance/deployment/src/test/java/io/quarkus/smallrye/faulttolerance/test/ObjectBeanTest.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.smallrye.faulttolerance.test;
+
+import javax.enterprise.inject.Produces;
+import javax.inject.Named;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+/**
+ * Verify that Fault Tolerance does not break if there is a CDI bean without a superclass
```

Comment:
There's no FT in the test at all?

Reply:
I guess it's about the `object` being processed by the `FTProcessor`, right? Gotcha. It looks weird, but it's probably enough.

Reply:
Yep, this is enough to reproduce the original issue if I deliberately remove the fix

Reply:
Nice, thanks!